pkgs/applications/blockchains/bitcoin.nix

```diff
@@ -36,15 +37,16 @@ stdenv.mkDerivation rec {
       "https://bitcoincore.org/bin/bitcoin-core-${version}/bitcoin-${version}.tar.gz"
       "https://bitcoin.org/bin/bitcoin-core-${version}/bitcoin-${version}.tar.gz"
     ];
-    sha256 = "4bbd62fd6acfa5e9864ebf37a24a04bc2dcfe3e3222f056056288d854c53b978";
+    sha256 = "1a91202c62ee49fb64d57a52b8d6d01cd392fffcbef257b573800f9289655f37";
```

Comment:
This hash matches the value signed by `Bitcoin Core binary release signing key` at https://bitcoincore.org/bin/bitcoin-core-0.21.0/SHA256SUMS.asc and I have verified the signature in that file.